src/FormControl/_FormControlCaption.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import InputCaption from '../internal/components/InputCaption'
 import {SxProp} from '../sx'
-import {FormControlContext} from './FormControl'
+import {FormControlContext} from './_FormControlContext'
 
 const FormControlCaption: React.FC<React.PropsWithChildren<{id?: string} & SxProp>> = ({children, sx, id}) => {
-  const {captionId, disabled} = React.useContext(FormControlContext)
+  const {captionId, disabled} = React.useContext(FormControlContext) ?? {}
```

Comment:
The default value for this context is now `null` so that `useForwardedProps` can handle things differently depending on whether or not the context is available